c++/src/kj/compat/http-test.c++

```diff
@@ -5200,6 +5200,37 @@ KJ_TEST("HttpClient concurrency limiting") {
   KJ_EXPECT(callbackEvents == kj::ArrayPtr<const CallbackEvent>({ {0, 0} }));
 }
 
+KJ_TEST("NetworkHttpClient connect impl") {
+  {
+    KJ_HTTP_TEST_SETUP_IO;
+  }
```

Comment:
Since this test relies on having access to the network, I think it should follow the same pattern as the ones below it:
```
#if KJ_HTTP_TEST_USE_OS_PIPE
// A comment that this test relies on having access to a real network.
KJ_TEST(...) {
  auto io = kj::setupAsyncIo();
  // ...
```